### PR TITLE
This Month in Xilem, July 2024

### PR DESCRIPTION
**Pull request by DJMcNab**
These are the updates which happened in July 2024

### CODE REVIEW

**Comment by PoignardAzur** [comment]
Will look over this tomorrow.

**Comment by jaredoconnell** [comment]
Is it worth mentioning the new Xilem example? That landed in July.

**Comment by davelab6** [comment]
I'm interested in a comparison of the roadmap to the actuals, and the next short term roadmap :)

File: content/blog/2024-07-31-tmix-07.md
Comment (DJMcNab) [inline content/blog/2024-07-31-tmix-07.md:6]: I'm not sure that this sends out the right message, because we are still a community organisation, with open membership. Admittedly, the dynamics here are quite weird, due to our "personal" roadmaps and being full-time.

I do agree that Philipp's work has been excellent, and should be called out again. I'd suggest a sentence or two in the Xilem section for that.

(Bruce's work has also been extremely helpful, both in the visible changes being made and in performing reviews, and should be mentioned somewhere.)

File: content/blog/2024-07-31-tmix-07.md
Comment (DJMcNab) [inline content/blog/2024-07-31-tmix-07.md:13]: This sentence could do with being rewritten.

File: content/blog/2024-07-31-tmix-07.md
Comment (DJMcNab) [inline content/blog/2024-07-31-tmix-07.md:14]: The calculator example can also go here. That might be worth a screenshot? We want to aim for at least one screenshot in each tmix now, as far as I know.

File: content/blog/2024-07-31-tmix-07.md
Comment (DJMcNab) [inline content/blog/2024-07-31-tmix-07.md:28]: We should mention something about this demo being created with Xilem Web, and probably have a screenshot of it.

**Review by DJMcNab — COMMENTED** [review]
Some light tweaks. I can't approve as I'm officially the author, although this is in draft anyway.

**Comment by PoignardAzur** [comment]
I think this is ready to publish. I'll hit the merge button this evening if there's no pressing concern.

File: content/blog/2024-07-31-tmix-07.md
Comment (xStrom) [inline content/blog/2024-07-31-tmix-07.md:37]:
```suggestion
  Note that the interactive demo is built with `xilem_web`.
```

File: content/blog/2024-07-31-tmix-07.md
Comment (jaredoconnell) [inline content/blog/2024-07-31-tmix-07.md:28]: I think this belongs in the Xilem section. The old Calculator example was pure Masonry. The new one is Xilem.

**Review by jaredoconnell — APPROVED** [review]

File: content/blog/2024-08-08-tmix-07/index.md
Comment (PoignardAzur) [inline content/blog/2024-08-08-tmix-07/index.md]:
```suggestion
- [rfcs#7][] specifies a clearer architecture for passes, including simplifications necessary for compositor use.
```

Comment (DJMcNab) [reply content/blog/2024-08-08-tmix-07/index.md]: This looks like a duplicate of de71af3ffa12f8fbeaa60430785aa2f40f9e19cf?

**Review by xStrom — APPROVED** [review]

**Pull request merged**
